test/js/reducers/questionnaire.spec.js

```diff
@@ -285,7 +285,7 @@ describe('questionnaire reducer', () => {
   })
 })
 
-const questionnaire = {
+const questionnaire = () => ({
```

Comment:
(Out of curiosity) Why this? The only reason I can think of is immutability, is that it?